.changeset/tame-ligers-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cli": major
+---
+
+Api generator is now part of `@comet/cli` package and can be run with `comet api-generator` command.
```

Comment:
We should have an upgrade script for this.

Reply:
i know, already on it.

Reply:
https://github.com/vivid-planet/comet-upgrade/pull/65